src/debugger/sourceMap.ts

```diff
@@ -67,7 +69,11 @@ export class SourceMapUtil {
 
             if (sourceMap.sources) {
                 sourceMap.sources = sourceMap.sources.map(sourcePath => {
-                    return this.updateSourceMapPath(sourcePath, sourcesRootPath);
+                    if (IS_REMOTE.test(sourcePath) || path.isAbsolute(sourcePath)) {
```

Comment:
Actually after rethinking this a bit I propose just to test whether sourcePath is remote URL and skip update in this case, and in all other case just feed it into update. I feel that rewriting paths becomes too complicated and probably needs a complete rewrite, probably leveraging new node2-debug features (such as remapping)